### DISCUSSION TITLE
[New Feature] Option to prevent aircraft from landing on the map

### DISCUSSION THREAD

**Discussion by Crimsonum** [Vinifera-Developers/Vinifera#1503] [Feature Requests]
### Description:
When giving a move order to aircraft, they will always land at the designated spot. This is in many cases undesirable, as it prevents aircraft from passively engaging enemies, and makes them less maneuverable. Having the option to prevent aircraft from landing anywhere but helipads would greatly improve air-to-surface combat (and air-to-air, if implemented). This is the default behavior in RA2 and later games.

### Possible Implementation:
New key for AircraftTypes: `HelipadBound=` (bool). If set, the aircraft cannot land outside helipads. Another key could control whether or not the aircraft should crash if no helipads are present.

There's a boolean key called `Landable` for AircraftTypes, but it doesn't actually prevent the aircraft from landing if disabled, and disabling it causes problems when producing the aircraft.

**Comment by XNiPERIANZ** [comment]
in ts vanilla [landable] tag work as intended ... after using tspacth/vinifera this tag no longer work.  the [landable] tag wont give you options to build aircraft when there's no free helipad to dock. Also in RA2, aircraft can be build from different types factory as example rocketer from barrack , kirov from warfactory and last from helipad its self, maybe add multiple dock number in single helipad would very help for modding ts.
